metalog.conf

```diff
@@ -99,6 +99,34 @@ SSH Server:
     logdir   = "/var/log/sshd"
     break    = 1
 
+# why the hell do you still run that thing !?
+Telnet:
+    program  = "login"
+    logdir   = "/var/log/telnet"
+    break    = 1
+
+Imap:
+     program  = "/usr/sbin/imapd"
+     logdir   = "/var/log/imap"
+     break    = 1
+
+POP Toaster:
+    program  = "/usr/sbin/ipop3d"
+    logdir   = "/var/log/pop"
+    break    = 1
+
+Mail:
```

Comment:
You didn't remove the Mail section, so this would be a duplicate. The POP Toaster section had commented SMTP relaying commands in it that are missing here.